From 9210ea52d747d19f9f8925575252d93200a260ab Mon Sep 17 00:00:00 2001
From: KanadeAlt <enashinonome264@gmail.com>
Date: Mon, 29 Sep 2025 19:51:36 +0000
Subject: [PATCH 1/4] Revert "common: Unset MSM_VIDC_TARGET_LIST"

This reverts commit cd562c511813e12a60f5c8e5a2d755647214035c.
---
 BoardConfigQcom.mk | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/BoardConfigQcom.mk b/BoardConfigQcom.mk
index b40f592..95c73ae 100644
--- a/BoardConfigQcom.mk
+++ b/BoardConfigQcom.mk
@@ -349,18 +349,25 @@ $(call soong_config_set,qtidisplay,gralloc_handle_has_reserved_size,$(TARGET_GRA
 $(call soong_config_set,qtidisplay,gralloc_handle_has_ubwcp_format,$(TARGET_GRALLOC_HANDLE_HAS_UBWCP_FORMAT))
 
 ifneq ($(filter $(UM_3_18_HAL_FAMILY),$(TARGET_BOARD_PLATFORM)),)
+    MSM_VIDC_TARGET_LIST := $(UM_3_18_HAL_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8996
 else ifneq ($(filter $(UM_4_9_LEGACY_FAMILY),$(TARGET_BOARD_PLATFORM)),)
+    MSM_VIDC_TARGET_LIST := $(UM_4_9_LEGACY_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8953
 else ifneq ($(filter $(UM_4_4_HAL_FAMILY),$(TARGET_BOARD_PLATFORM)),)
+    MSM_VIDC_TARGET_LIST := $(UM_4_4_HAL_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8998
 else ifneq ($(filter $(UM_4_19_LEGACY_FAMILY),$(TARGET_BOARD_PLATFORM)),)
+    MSM_VIDC_TARGET_LIST := $(UM_4_19_LEGACY_FAMILY)
     QCOM_HARDWARE_VARIANT := sdm660
 else ifneq ($(filter $(UM_4_9_FAMILY),$(TARGET_BOARD_PLATFORM)),)
+    MSM_VIDC_TARGET_LIST := $(UM_4_9_FAMILY)
     QCOM_HARDWARE_VARIANT := sdm845
 else ifneq ($(filter $(UM_4_14_FAMILY),$(TARGET_BOARD_PLATFORM)),)
+    MSM_VIDC_TARGET_LIST := $(UM_4_14_FAMILY)
     QCOM_HARDWARE_VARIANT := sm8150
 else ifneq ($(filter $(UM_4_19_FAMILY),$(TARGET_BOARD_PLATFORM)),)
+    MSM_VIDC_TARGET_LIST := $(UM_4_19_FAMILY)
     QCOM_HARDWARE_VARIANT := sm8250
 else ifneq ($(filter $(UM_5_4_FAMILY),$(TARGET_BOARD_PLATFORM)),)
     QCOM_HARDWARE_VARIANT := sm8350
@@ -377,6 +384,7 @@ else ifneq ($(filter $(UM_6_6_FAMILY),$(TARGET_BOARD_PLATFORM)),)
         QCOM_HARDWARE_VARIANT := sm8750
     endif
 else
+    MSM_VIDC_TARGET_LIST := $(TARGET_BOARD_PLATFORM)
     QCOM_HARDWARE_VARIANT := $(TARGET_BOARD_PLATFORM)
 endif
 
-- 
2.53.0.windows.1

